budgie-hotcorners/src/budgie-hotcorners-settingswindow.vala

```diff
@@ -0,0 +1,779 @@
+                    return;
+                }
+                update_settings_fromdelay();
+            });
+            pressure_slider.value_changed.connect(()=> {
+                if (!try_block_settingsflow(SettingsFlow.WIDGETSTOSETTINGS)) {
+                    return;
+                }
+                update_settings_frompressure();
+            });
+            this.show_all();
+            /*
+            to set left-right space, we need allocated space of the window.
+            so, show first, then calculate, show again.
+            */
+            int w; int h;
+            this.get_size(out w, out h);
+            /* left/right, top, bottom, inner-hor, inner-vert */
+            make_bulletsection((int)(w - 185)/2, 15, 50, 10, 2);
+
+            set_interface_sensitive(newhcornersettings.get_boolean("active"));
+            masterswitch.state_set.connect((newstate)=> {
+                set_interface_sensitive(newstate);
+                return false;
+            });
+
+            this.show_all();
+            if (try_block_settingsflow(SettingsFlow.SETTINGSTOWIDGETS)) {
+                /*
+                pretty late, because of gui appearance on startup (sliding
+                pages). since it is after the connect(), block loopback.
+                */
+                update_commands_fromsettings();
+            }
+            pages.set_visible_child_name("empty_page");
+            this.destroy.connect(Gtk.main_quit);
+        }
+
+        public void initialiseLocaleLanguageSupport() {
+            GLib.Intl.setlocale(GLib.LocaleCategory.ALL, "");
+            GLib.Intl.bindtextdomain(
+                Config.GETTEXT_PACKAGE, Config.PACKAGE_LOCALEDIR
+            );
+            GLib.Intl.bind_textdomain_codeset(
+                Config.GETTEXT_PACKAGE, "UTF-8"
+            );
+            GLib.Intl.textdomain(Config.GETTEXT_PACKAGE);
+        }
+
+        private void activate_byname (string name) {
+            if (!service_ison("org.UbuntuBudgie.ShufflerInfoDaemon")) {
+                return;
+            }
+            try {
+                shufflerinfoclient.ActivateWindowByname(name);
+            }
+            catch (Error e) {
+                stderr.printf ("%s\n", e.message);
+            }
+        }
+
+        private void get_shuffler_client () {
+            try {
+                shufflerinfoclient = Bus.get_proxy_sync (
+                    BusType.SESSION, "org.UbuntuBudgie.ShufflerInfoDaemon",
+                    ("/org/ubuntubudgie/shufflerinfodaemon")
+                );
+            }
+            catch (Error e) {
+                stderr.printf ("%s\n", e.message);
+            }
+        }
+
+        void setup_dbus_server () {
+            Bus.own_name (
+                BusType.SESSION, "org.UbuntuBudgie.HotCornersControlRuns",
+                BusNameOwnerFlags.NONE, on_bus_acquired,
+                () => {},
+                () => stderr.printf ("Could not acquire name\n")
```

Comment:
debug warning type message?

Reply:
Seems we can only use std or stderr output here.